openapi/components/schemas/Rules/RulesEmailNotification.yaml

```
@@ -35,7 +35,7 @@ properties:
       If your email message templates are localized into more than one language,
       set a customer locale.
 
-      Invalid placeholders render as an empty strings. 
+      Invalid placeholders render as an empty strings.
```

nitpick: `an` is not needed here
```suggestion
      Invalid placeholders render as empty strings.
```